pgoapi/pgoapi.py

```diff
@@ -338,18 +373,17 @@ def spin_nearest_fort(self):
         if destinations:
             nearest_fort = destinations[0][0]
             nearest_fort_dis = destinations[0][1]
-            self.log.info('Nearest fort distance is %s', nearest_fort_dis)
+            self.log.info('[STANDING]\t- Nearest PokeStop is %sm away', int(nearest_fort_dis))
 
             # Fort is close enough to change our route and walk to
             if nearest_fort_dis > 40.00 and nearest_fort_dis <= 100:
                 lat = nearest_fort['latitude']
-                long = nearest_fort['longitude']
-                self.walk_to_fort(destinations[0], directly=True)
+                lon = nearest_fort['longitude']
+                self.walk_to_fort(destinations[0][0], directly=False)
```

Comment:
I'm a complete noob! I've been messing around with a whole lot of the code just to see what happens. I should probably check through everything before I commit; this is certainly not ready to roll out code!